x-pack/plugins/ml/common/types/ml_url_generator.ts

```diff
@@ -167,6 +167,7 @@ export interface DataFrameAnalyticsExplorationQueryState {
   ml: {
     jobId: JobId;
     analysisType: DataFrameAnalysisConfigType;
+    isTraining?: string;
```

Comment:
should it be a `boolean` instead? 🤔

Reply:
Yes, good catch. Updated so param is always a boolean. 43a33e33877d0893f3f0041df1c7240c1df45860